packages-common/nanoflow-commons/src/external/CallPhoneNumber.ts

```diff
@@ -8,14 +8,13 @@ import ReactNative from "react-native";
 
 /**
  * @param {string} phoneNumber - This field is required.
- * @returns {string}
+ * @returns {Promise.<boolean>}
  */
-// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
-function CallPhoneNumber(phoneNumber?: string): Promise<boolean> {
+export async function CallPhoneNumber(phoneNumber?: string): Promise<boolean> {
     // BEGIN USER CODE
 
     if (!phoneNumber) {
-        throw new TypeError("Input parameter 'Phone number' is required");
+        return Promise.reject(new TypeError("Input parameter 'Phone number' is required"));
```

Comment:
We should never use `TypeError ` just use Error for all
```suggestion
        return Promise.reject(new Error("Input parameter 'Phone number' is required"));
```

Reply:
Done